src/utils/sortObjectKeys.ts

```diff
@@ -0,0 +1,23 @@
+import { Settings } from "../types/settings";
+
+function sortObjectKeys(
+  obj: any,
+  sortOrder: Settings["sort"]
+): any {
+  if (sortOrder === "none") return obj;
+  if (Array.isArray(obj)) {
+    return obj.map(item => sortObjectKeys(item, sortOrder));
+  } else if (obj !== null && typeof obj === "object") {
+    const keys = Object.keys(obj);
+    const sortedKeys =
+      sortOrder === "A-Z" ? keys.sort() : keys.sort().reverse();
+    const sortedObj: any = {};
+    for (const key of sortedKeys) {
+      sortedObj[key] = sortObjectKeys(obj[key], sortOrder);
+    }
+    return sortedObj;
```

Comment:
Logic for sorting can be abstracted and used based on the sortOrder type, this because in a future we could have more options than simply alphabetically and alphabetically reverse.